bucket/TencentMeeting.json

```
@@ -0,0 +1,33 @@
+{
+    "version": "3.3.4.444",
+    "description": "Video conferencing app by Tencent",
+    "homepage": "https://meeting.tencent.com/",
+    "license": {
+        "identifier": "EULA",
+        "url": "https://meeting.tencent.com/declare.html"
+    },
+    "notes": "We don't persist your Tencent Meeting data, they are still storaged in \"$env:APPDATA\\Tencent\\WeMeet\".",
+    "url": "https://updatecdn.meeting.qq.com/cos/ce515ae2a884821b62f94cb0b24a68cd/TencentMeeting_0300000000_3.3.4.444.publish.exe#/dl.7z",
+    "hash": "b1b50419765c311333486414f478ed073a08776f19fc4fac0d1a42f0e1e61c98",
+    "installer": {
+        "script": [
+            "Rename-Item $dir\\$_*_ -NewName $version",
+            "Remove-Item \"$dir\\$*\" -Recurse"
+        ]
+    },
```

`$version`  在 `installer` 不工作。
```suggestion
    "pre_install": "Rename-Item -Path \"$dir\\`$_*\" -NewName \"$version\" -Force",
    "post_install": "Remove-Item \"$dir\\$*\" -Recurse",
```

---

我安装了好几次   $version起作用的啊